inst/extdata/variable_details.csv

```diff
@@ -2160,6 +2170,35 @@ number_conditions,number_conditions_cat6_2,cat,"cchs2001_p, cchs2003_p, cchs2005
+OH2_30,OH2_30_cat6_2,cat,"cchs2003_i, cchs2005_i, cchs2009_20010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2017_2018_i","cchs2003_i::OH2C_30, cchs2005_i::OH2E_30, cchs2017_2018_i::DEN_101, [OH2_30]",ICES,cat,2,6,Twice/Day,Twice/Day,N/A,2,Twice/Day,Freq. brushing teeth,Frequency of brushing teeth,,3.0.0,lastUpdated,active,,
+OH2_30,OH2_30_cat6_3,cat,"cchs2003_i, cchs2005_i, cchs2009_20010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2017_2018_i","cchs2003_i::OH2C_30, cchs2005_i::OH2E_30, cchs2017_2018_i::DEN_101, [OH2_30]",ICES,cat,3,6,Once/Day,Once/Day,N/A,3,Once/Day,Freq. brushing teeth,Frequency of brushing teeth,,3.0.0,lastUpdated,active,,
+OH2_30,OH2_30_cat6_4,cat,"cchs2003_i, cchs2005_i, cchs2009_20010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2017_2018_i","cchs2003_i::OH2C_30, cchs2005_i::OH2E_30, cchs2017_2018_i::DEN_101, [OH2_30]",ICES,cat,4,6,< Once/Day,< Once/Day,N/A,4,< Once/Day,Freq. brushing teeth,Frequency of brushing teeth,,3.0.0,lastUpdated,active,,
+OH2_30,OH2_30_cat6_5,cat,"cchs2003_i, cchs2005_i, cchs2009_20010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2017_2018_i","cchs2003_i::OH2C_30, cchs2005_i::OH2E_30, cchs2017_2018_i::DEN_101, [OH2_30]",ICES,cat,5,6,Once/Week,Once/Week,N/A,5,Once/Week,Freq. brushing teeth,Frequency of brushing teeth,,3.0.0,lastUpdated,active,,
+OH2_30,OH2_30_cat6_6,cat,"cchs2003_i, cchs2005_i, cchs2009_20010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2017_2018_i","cchs2003_i::OH2C_30, cchs2005_i::OH2E_30, cchs2017_2018_i::DEN_101, [OH2_30]",ICES,cat,6,6,< Once/Week,< Once/Week,N/A,6,< Once/Week,Freq. brushing teeth,Frequency of brushing teeth,,3.0.0,lastUpdated,active,,
+OH2_30,OH2_30_cat6_NA::a,cat,"cchs2003_i, cchs2005_i, cchs2009_20010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2017_2018_i","cchs2003_i::OH2C_30, cchs2005_i::OH2E_30, cchs2017_2018_i::DEN_101, [OH2_30]",ICES,cat,NA::a,6,not applicable,not applicable,N/A,96,not applicable,Freq. brushing teeth,Frequency of brushing teeth,,3.0.0,lastUpdated,active,,
+OH2_30,OH2_30_cat6_NA::b,cat,"cchs2003_i, cchs2005_i, cchs2009_20010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2017_2018_i","cchs2003_i::OH2C_30, cchs2005_i::OH2E_30, cchs2017_2018_i::DEN_101, [OH2_30]",ICES,cat,NA::b,6,missing,missing,N/A,"[97,99]",don't know (97); refusal (98); not stated (99),Freq. brushing teeth,Frequency of brushing teeth,,3.0.0,lastUpdated,active,,
+OH2_30,OH2_30_cat6_NA::b,cat,"cchs2003_i, cchs2005_i, cchs2009_20010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2017_2018_i","cchs2003_i::OH2C_30, cchs2005_i::OH2E_30, cchs2017_2018_i::DEN_101, [OH2_30]",ICES,cat,NA::b,6,missing,missing,N/A,else,else,Freq. brushing teeth,Frequency of brushing teeth,,3.0.0,lastUpdated,active,,

```

Comment:
@caitlink12 Can you do the same as in [this comment](https://github.com/Big-Life-Lab/cchsflow/pull/140/commits/db122101b4616be897bba7504d1a33352759da97#r2291391462)